armi/reactor/reactors.py

```diff
@@ -2129,6 +2129,77 @@ def growToFullCore(self, cs):
 
         return converter
 
+    def checkGridPlateMaterialConsistency(self):
+        """check that the grid plates in the core are the same material"""
+        gridPlateBlocks = self.getBlocks(Flags.GRID_PLATE)
+        for ib, b in enumerate(gridPlateBlocks):
+            gridPlateComponent = b.getComponent(Flags.GRID_PLATE)
+            gridPlateMaterial = gridPlateComponent.material
+            if ib == 0:
+                refGridPlateMat = gridPlateMaterial
+                continue
```

Comment:
Maybe just get this out of the loop and have `refGridPlateMat = self.getFirstBlock(Flags.GRID_PLATE).getComponent(Flags.GRID_PLATE).material`?

Reply:
Addressed https://github.com/terrapower/armi/pull/993/commits/01ce852e7d7759ae463e6f3c612e031db1f77717